data/crawler.py

```diff
@@ -277,13 +277,29 @@ def get_opendata_sections():
         with open(filename, 'w') as f:
             json.dump(sections, f)
 
+def get_scholarships():
+    url = ('https://api.uwaterloo.ca/v2/awards/undergraduate.json?'
+          'key={api_key}'.format(
+              api_key = s.OPEN_DATA_API_KEY
+              )
+          )
+    response = requests.get(url).text
+
+    scholarship_data = json.loads(response)
+
+    filename = os.path.join(os.path.dirname(__file__),
+            '%s/scholarships.json' % c.SCHOLARSHIPS_DATA_DIR)
```

Comment:
nit: Prefer consistent string formatting/interpolation (above, you use `''.format` which is fine, but here you're using `'%s'`). We've generally been using this one (`'%s'`).